doc/reference/configuration/configuration_reference.rst

```diff
@@ -799,15 +790,8 @@ URI parameters that can be used in the following options:
 
```

Comment:
Lines 782-783:

-   :ref:`**iproto_advertise**.\<peer_or_sharding\>.params <configuration_reference_iproto_advertise.peer_sharding.params>` - undescore after `iproto`
-   :ref:`iproto.listen <configuration_reference_iproto_listen>` - no `params` in the end

Reply:
Please check for `iproto_advertise` (with underscore) across the page, there are more occurrences.

Reply:
thanks, fixed in all places